### PR TITLE
feat: Splits and styles Nav and Zelda components

### PR DESCRIPTION
**Pull request by tribakzero**

### CODE REVIEW

File: session3/styledComponents/src/Nav.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { StyledNav } from './Nav.style.js'
+
+class Nav extends Component {
```
Comment (timgivois) [inline session3/styledComponents/src/Nav.js]: I don't see a use to this Component. It would be the same to just import `StyledNav` in `App.js`.

**Comment by timgivois** [comment]
You didn't complete the whole challenge. Refer to the slides :).

File: session3/styledComponents/src/Nav.js (hunk 1 above)
Comment (tribakzero) [reply session3/styledComponents/src/Nav.js]: Thanks @timgivois.
True, I was expecting it to grow bigger but also I moved the structure of the Header recently and it is in a kind of WIP.

**Comment by tribakzero** [comment]
@timgivois You're right, I just assumed  it was kind of easier, but I'll work on it. Thanks.

**Pull request closed without merging**